inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/activation.cpp

```diff
@@ -29,13 +29,30 @@ const std::vector<ActivationTypes> activationTypes = {
         Exp,
         Log,
         Sign,
-        Abs
+        Abs,
+        Clamp,
+        Negative,
+        Acos,
+        Asin,
+        Atan,
+        //Ceil,
```

Comment:
Why `ceil` is commented?

Reply:
Because the Ceiling test on CPU fails

Reply:
Please, use `disable_tests_config` for it and create a ticket